UnityProject/Assets/Scripts/HealthV2/Living/LivingHealthMasterBase.cs

```diff
@@ -1912,8 +1971,74 @@ public void ExposePressureTemperature(float EnvironmentalPressure, float Environ
 				}
 			}
 
-			healthStateController.SetTemperature(ExtremistTemperature);
-			healthStateController.SetPressure(ExtremistPressure);
+			if (ExtremistPressure != ExtremistPressureCash)
```

Comment:
this might be polluting the class unnecesarilly. Can't move the alerts to a dedicated class and make LHB use it (or even maybe the other way around). LHB has way too many concerns.

Reply:
humm, Where would be the best place to put it?

Reply:
it probably deserves to be its own class somewhere inside the UI namespace

Reply:
to be fair this is living on the player, and doesn't care if there's a player Connected or not